tests/storage/snapshot.go

```diff
@@ -1533,7 +1535,7 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				snapshot = libstorage.WaitSnapshotSucceeded(virtClient, vm.Namespace, snapshot.Name)
 				Expect(snapshot.Status.SnapshotVolumes.IncludedVolumes).Should(HaveLen(2))
 				Expect(snapshot.Status.SnapshotVolumes.IncludedVolumes[0]).Should(Equal("snapshotablevolume"))
-				Expect(snapshot.Status.SnapshotVolumes.IncludedVolumes[1]).Should(Equal(pvc.Name))
+				Expect(snapshot.Status.SnapshotVolumes.IncludedVolumes[1]).Should(Equal(fmt.Sprintf("persistent-state-for-%s", vm.Name)))
```

Comment:
in case the snapshot is online will the backend pvc be in the excludedVolumes?

Reply:
Nope since it's filtered before including/excluding. When writing the code there was some reason to do this but I can look into it.